### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by ashfurrow** [Moya/Harvey#7]
Every cool project needs a cool logo. I took a look at the [character Harvey](http://farscape.wikia.com/wiki/Harvey)'s outfit and came up with some ideas for an abstract representation. This is the same process @orta went through to design [Moya's original logo](https://github.com/Moya/Moya/pull/27/files). Here's what I've got, very open to suggestions.

<img width="858" alt="screen shot 2017-09-30 at 4 17 24 pm" src="https://user-images.githubusercontent.com/498212/31049143-e470a896-a5fa-11e7-9621-4846937e6462.png">

or

<img width="660" alt="screen shot 2017-09-30 at 4 17 28 pm" src="https://user-images.githubusercontent.com/498212/31049144-e75d1cec-a5fa-11e7-95b5-708a55064600.png">

or

<img width="929" alt="screen shot 2017-09-30 at 4 17 36 pm" src="https://user-images.githubusercontent.com/498212/31049148-e9ec941a-a5fa-11e7-86c9-c4cfcce13145.png">

The blue is the same blue hue as Moya's logo, but I'm totally open to any colour scheme.

**Comment by sunshinejr** [comment]
@ashfurrow I like these in blue (2 & 3)! How would it look smaller in width? Also, do you think we could add a name somewhere in the logo or is it too much? 🤔

**Comment by ashfurrow** [comment]
I generally don't like names in logos, like a name _as_ a logo like Danger's:

![danger-logo-small 2x-82652605](https://user-images.githubusercontent.com/498212/31082086-e6fede18-a75b-11e7-8359-5a0e4f4bdbad.png)

But a name within a logo might work too. I'll try and brainstorm.

**Comment by orta** [comment]
You could probably play with a "H" - then try re-use the soya color scheme to hint at a connection

**Comment by orta** [comment]
lols, try make a H from the Moya shapes :P

**Comment by SD10** [comment]
![screen shot 2017-10-02 at 2 06 37 pm](https://user-images.githubusercontent.com/7445580/31094458-f77c2aa6-a77a-11e7-9357-fbbf888dae75.png)

You could make an H from inverting the middle of one Moya logo on top of another.

**Comment by ashfurrow** [comment]
I vectorized and played around with @SD10 's idea above, what do we think of this?

![logo](https://user-images.githubusercontent.com/498212/31735454-fa912830-b40f-11e7-802f-f857a86c2f70.png)

![logo2](https://user-images.githubusercontent.com/498212/31735465-014899f6-b410-11e7-9df7-292f66629ed8.png)

Happy to accept feedback. The Sketch file is here: 

[Harvey.sketch.zip](https://github.com/Moya/Harvey/files/1395838/Harvey.sketch.zip)

**Comment by SD10** [comment]
My only problem with my own suggestion is I didn't use all 3 of Moya's colors 🤔

**Comment by ashfurrow** [comment]
Maybe add a blue line each on the left and right of the main column? I'll play with it and update 👍

**Comment by mikesanderson** [comment]
Placing the H in the negative space is an interesting idea. This is colorful too, never noticed Moya had such a nice color pallet. 

Logo's should connect with the meaning of what they stand for in some way. Harvey is a network stubbing library? 

An idea building on this last one: Take an "H" in a sans-serif font, a font suitable to write the name Harvey in. Arrange the Colors/Shapes around that, or arrange the colors shapes then place the the "H" over it to mask. Then put the word "Harvey" to the right with the same capital H. (How logo & name look together is a consideration anyway). The logo would still work independently, the H just needs a little more definition anyway. The idea is, Harvey fills in the gap.

**Comment by mikesanderson** [comment]
Using @ashfurrow as a launching off point I went through some iterations and came up with this: 

![logo_mike](https://user-images.githubusercontent.com/6426698/31782723-ddea4ba2-b4c9-11e7-9714-a4acbdde667a.png)

Sketch file is 
[Harvey_logo_2.sketch.zip](https://github.com/Moya/Harvey/files/1399258/Harvey_logo_2.sketch.zip)
, Sorry for the atrocious techniques used I'm sure, I cobbled this out (also why imprecise alignment--no need to do perfect until final decisions made). The logo iterations are grouped. Also, this sketch file has "Document Colors" from the Moya logo, really convenient, though that only took a few minutes. 

The evolution: 

<img width="655" alt="screen shot 2017-10-19 at 12 35 19 pm" src="https://user-images.githubusercontent.com/6426698/31783052-e5203822-b4ca-11e7-8b6c-2e22c99d8187.png">

On the final evolution, shape exact positioning needs adjustment, which should be after selection of font (Any comments on that?). Looks composed of horizontal and vertical lines, but by connecting points will be able to see shapes of diamonds. For example, the top and bottom blue corner. 

The two matching shapes on the side enclose the inner area, suggesting containment. Being very symmetrical helps this. But they are hands off. 

The middle part then forms the bulk of it, formed by two symmetrical shapes that resemble breakpoints (not directly related to Harvey but being an IDE/debugging shape gets there). They also point up and down, evoking the up/down of server requests. Things going up and down in a controlled, symmetrical way.

Colors could be adjusted, the green or purple. Still, at only 8 shapes the logo is simple. Feedback welcome.
